src/main/java/com/passninja/model/Pass.java

```diff
@@ -98,6 +98,18 @@ public static PassninjaResponse<Pass> get(String passType, String serialNumber)
         return request(RequestMethod.GET, RESOURCE + "/" + passType + "/" + serialNumber, null, Pass.class, null);
     }
 
+    public static PassninjaResponse<Pass> find(String passType) throws ApiException, IOException,
```

Comment:
looks like this method returns a single pass, am I right?

Reply:
it should return all the passes issued for that passType... are you not seeing that?

Reply:
do you know how to make this method return an array of `Pass`es ?

Reply:
Hi Richard! Yes, I know, if you grant me permissions I could push the required changes.

Reply:
@inginiir sent you repo invite just now... let me know if you don't see it:

https://github.com/flomio/passninja-java/invitations